terraform/modules/concourse_worker/iam.tf

```diff
@@ -55,6 +55,31 @@ resource "aws_iam_policy" "concourse_parameters_worker" {
   policy      = data.aws_iam_policy_document.concourse_parameters_worker.json
 }
 
+data "aws_iam_policy_document" "concourse_secrets_read" {
+  statement {
+    actions = [
+      "secretsmanager:DescribeSecret",
+      "secretsmanager:GetSecretValue",
+      "secretsmanager:ListSecretVersionIds"
```

Comment:
although not required, it is better for git if last item in list has a trailing comma when the language allows it and HCL does